recipes/greedy/build.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+
+mkdir build
+cd build
+
+export GIT_DISCOVERY_ACROSS_FILESYSTEM=1
+
+export CFLAGS="${CFLAGS} -I ${PREFIX}/include/eigen3"
+export CXXFLAGS="${CXXFLAGS} -I ${PREFIX}/include/eigen3"
+
+
+cmake  $CMAKE_ARGS ../greedy \
+      -DITK_DIR="${CONDA_PREFIX}/lib/cmake/ITK" \
```

Comment:
```suggestion
      -DITK_DIR="${PREFIX}/lib/cmake/ITK" \
```
$CONDA_PREFIX is not the correct location to look for build dependencies. These dependencies should always be loaded from $PREFIX which is the requirements.host prefix during the build.